docs/docs/self-hosting/index.md

```diff
@@ -5,20 +5,32 @@ description: Getting started self hosting Ente Photos and/or Ente Auth
 
 # Self Hosting
 
-The entire source code for Ente is open source, including the servers. This is
+The entire source code for Ente is open source,
+[including the servers](https://ente.io/blog/open-sourcing-our-server/). This is
 the same code we use for our own cloud service.
 
-> [!TIP]
->
-> You might find our [blog post](https://ente.io/blog/open-sourcing-our-server/)
-> announcing the open sourcing of our server useful.
+## Requirements
 
-## System requirements
+### Hardware
 
-The server has minimal resource requirements, running as a lightweight Go
-binary. It performs well on small cloud instances, old laptops, and even
+The server is capable of running on minimal resource requirements as a
+lightweight Go binary, since most of the intensive computational tasks are done
+on the client. It performs well on small cloud instances, old laptops, and even
 [low-end embedded devices](https://github.com/ente-io/ente/discussions/594).
 
+### Software
+
+#### Operating System
+
+Any Linux or \*nix operating system, Ubuntu or Debian is recommended to have a
+good Docker experience. Non-Linux operating systems tend to provide poor
+experience with Docker and difficulty with troubleshooting and assistance.
```

Comment:
I feel we should be crisp about our requirements and keep the home page as short as possible. The idea is that anyone can run `quickstart.sh` and if they run into trouble read more.

We could have a separate page for requirements.